internal/cluster/distribution/eks/eksadapter/cluster_manager.go

```diff
@@ -28,23 +28,33 @@ import (
 
 type clusterManager struct {
 	workflowClient client.Client
+	enterprise     bool
 }
 
 // NewClusterManager returns a new eks.ClusterManager
 // that manages clusters asynchronously via Cadence workflows.
-func NewClusterManager(workflowClient client.Client) eks.ClusterManager {
+func NewClusterManager(workflowClient client.Client, enterprise bool) eks.ClusterManager {
 	return clusterManager{
 		workflowClient: workflowClient,
+		enterprise:     enterprise,
 	}
 }
 
-func (n clusterManager) UpdateCluster(
+func (m clusterManager) UpdateCluster(
 	ctx context.Context,
 	c cluster.Cluster,
 	clusterUpdate eks.ClusterUpdate,
 ) error {
+	taskList := "pipeline"
+	workflowName := eksworkflow.UpdateClusterWorkflowName
+
+	if m.enterprise {
+		taskList = "pipeline-enterprise"
+		workflowName = "eks-update-cluster"
```

Comment:
Why is this different?

Reply:
We have to use "-v2" in pipeline, since the previous is already taken, but we can change to "-v2" in enterprise as well.

Reply:
How about renaming the old one to something else?

Reply:
If that doesn't break backward compatibility I'm happy to rename it (since probably this will be the official update workflow and that one will be deprecated).